### ISSUE TITLE
About page is a bit light

### ISSUE THREAD

**Issue by drnw** [tilix/tilix.github.io#7]
Does it need a few more words and perhaps a central image like [Mission](https://tilix.github.io/mission/)

**Comment by paulohagan** [comment]
Yes this page is very light and would be great to add some client logos across the footer and a call to action leading to form on contact page - see the sketch.
[Tilix-About.pdf](https://github.com/tilix/tilix.github.io/files/1221463/Tilix-About.pdf)

**Comment by drnw** [comment]
@paulohagan mixed up the [Clients](https://tilix.github.io/clients/) with [About](https://tilix.github.io/about) :-0

The suggestion is good improving [About](https://tilix.github.io/about) is more pressing.
